homeassistant/helpers/entity_component.py

```diff
@@ -30,21 +30,24 @@ class EntityComponent(object):
     """Helper class that will help a component manage its entities."""
 
     def __init__(self, logger, domain, hass,
-                 scan_interval=DEFAULT_SCAN_INTERVAL, group_name=None):
+                 scan_interval=DEFAULT_SCAN_INTERVAL, group_name=None,
+                 scan_sequential=False):
```

Comment:
Shouldn't be here. Entity Component can have any number of platforms which all have different implementations. No need to have a default.

Reply:
It's for the default that is used on stuff like input_*

Reply:
I don't think that the entity component should be aware at all. Only the EntityPlatform should be. We should have it be derived from the platform as a constant or from the entity based on functions.

Reply:
:+1: